_posts/2020-04-17-snowplow-amp-tracker-1-0-0-released/index.md

```diff
@@ -0,0 +1,186 @@
+
+Scroll percentage and engaged time metrics can be calculated by aggregating on these values. Additionally, this method can be used in combination with the new AMP web page entity to aggregate to a page view level.
+
+Documentation can be found [here][amp-ping-docs]
+
+<h3 id="wp-entity"> 1.2 Web Page Entity</h3>
+
+The web page entity has been introduced, and is attached to every event by default. The AMP web page entity attaches an AMP-specific entity to all events, against the [AMP web page schema][amp-wp-schema]. This contains the ampPageViewId.
+
+Note that the value provided is the AMP-provided [Page View ID 64](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#page-view-id-64):
+
+> Provides a string that is intended to be random with a high entropy and likely to be unique per URL, user and day.
+
+Therefore, aggregating on this value alone will not produce the same results as the javascript tracker page view id.
+
+<h3 id="modeling-pvs"> 1.3 Modeling Considerations</h3>
+
+The data provided on AMP differs from that of the Javascript tracker, and thus should be modeled differently.
+
+The scrollLeft and scrollTop fields provide the amount of scroll from the leftmost and topmost point of the page, in pixels. The scrollWidth/Height and viewportWidth/Height fields provide the size of the page, and size of the viewport, again in pixels. These values can therefore be aggregated together to calculate page scroll ratios in a similar but slightly different way to how this is done traditionally.
+
+The AMP Page View ID value provided is not unique to an instance of a page view, but rather is unique when combined with the URL, date, and AMP client ID, so data should be aggregated by the concatenation of these values, rather than the ID alone.
+
+
+<h2 id="user-journeys">1. Mapping User Journeys</h2>
+
+User identification using AMP is now significantly improved, and functionality has been introduced to allow users to track users across AMP and traditional web pages. The AMP ID entity facilitates identifying users who move from traditional pages to AMP, the AMP linker is now used to map users from AMP to traditional pages, and the AMP client ID is now attached to all events as distinct from other user ids.
+
+<h3 id="amp-id-entity"> 2.1 AMP ID Entity</h3>
+
+The AMP ID entity is a new feature to Snowplow tracking, with a specific purpose of making it easier to identify users on the AMP platform, and across AMP and non-AMP pages. By default, the AMP ID entity will be attached to all events, against the [AMP ID schema][amp-id-schema].
+
+This will contain the ampClientId (the consistent identifier for users on the AMP platform), the `user_id` if set, and the `domain_userid`.
```

Comment:
Is it worth calling out that the AMP tracking will never generate a domain_userid?